website/content/partials/known-issues/1_16-default-lcq-pre-1_9-upgrade.mdx

```diff
@@ -0,0 +1,26 @@
+### Default Lease Count Quota enabled when upgrading from Vault versions before 1.9
+
+#### Affected versions
+
+- 1.16.0
+
+#### Issue
+
+Default Lease Count Quotas are enabled for new clusters by consulting Vault's
+past versions.
+
+However, the version store was introduced in 1.9. As a result, upgrading from a
+version prior to 1.9 enables the Default Lease Count Quota.
+
+You can query the quota via the `/sys/quotas/lease-count/default` endpoint, and
+modify or delete it as appropriate:
```

Comment:
```suggestion
Before v1.9, Vault used a variety of ways to enable lease counting. Vault 1.9
introduced the version store, which standardized enforcement for lease count
quotas and enabled quotas by default. As a result, upgrading from a version
prior to 1.9 automatically enables the default lease count quota on your Vault
clusters.

You can review, delete, and modify the default lease count quotas with the
[`/sys/quotas/lease-count/default`](vault/api-docs/system/lease-count-quotas)
endpoint:
```

Style correction: write in active voice

Reply:
I believe I fixed this in the latest pass, but if you get another chance to look, I'd appreciate it!